src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/Resources/DeleteResourcesValidation.feature.disabled

```diff
@@ -0,0 +1,37 @@
+# This is a rough draft feature for future use.
+@ignore
+Feature: Resources "Delete" Operation validations
+
+        Background:
+            Given the Data Management Service must receive a token issued by "http://localhost"
+              And user is already authorized
+              And request made to "/ed-fi/absenceEventCategoryDescriptors" with
+                  """
+                    {
+                        "codeValue": "Sick Leave",
+                        "description": "Sick Leave",
+                        "effectiveBeginDate": "2024-05-14",
+                        "effectiveEndDate": "2024-05-14",
+                        "namespace": "uri://ed-fi.org/AbsenceEventCategoryDescriptor",
+                        "shortDescription": "Sick Leave"
+                    }
+                  """
+             Then it should respond with 201
+
+        @ignore
+        Scenario: Verify deleting a specific resource by ID
+             When a DELETE request is made to "/ed-fi/absenceEventCategoryDescriptors/{id}"
+             Then it should respond with 204
+
+        @ignore
+        Scenario: Verify error handling when deleting a non existing resource
+             When a DELETE request is made to "/ed-fi/absenceEventCategoryDescriptors/00112233445566"
+             Then it should respond with 400
+
+        @ignore
+        Scenario: Verify response code when GET a deleted resource
+            # The id value should be replaced with the resource created in the Background section
+             When a DELETE request is made to "/ed-fi/absenceEventCategoryDescriptors/{id}"
+             Then it should respond with 204
```

Comment:
It will be interesting to see if this will work, having two sets of "When/Then" blocks in one scenario. If it doesn't work, we can change to "Given a Delete request... When a get request ... Then ... 404".

Note that the response code will be 404, does not exist.

Reply:
Sure, yeah I was not sure about the 2 sets, but I saw some examples in the same way.